FlowCal/io.py

```diff
@@ -1160,11 +1163,88 @@ def hist_bin_edges(self, channels=None):
         # Get numerical indices of channels
         channels = self._name_to_index(channels)
 
-        # Get detector type of the specified channels
+        # Get resolution of the specified channels
         if hasattr(channels, '__iter__'):
-            return [self._hist_bin_edges[ch] for ch in channels]
+            return [self._resolution[ch] for ch in channels]
         else:
-            return self._hist_bin_edges[channels]
+            return self._resolution[channels]
+
+    def hist_bins(self, channels=None, nbins=None, log=False):
+        """
+        Get a set of histogram bins the specified channel(s).
+
+        These cover the range specified in ``FCSData.range(channels)`` with
+        a number of bins `nbins`, either linearly or logarithmically
+        spaced.
+
+        Parameters
+        ----------
+        channels : int, str, list of int, list of str
+            Channel(s) for which to get the histogram bins. If None, return
+            a list with bins for all channels, in the order of the
+            `channels` attribute.
+        nbins : int or list of ints, optional
+            The number of bins to calculate. If `channels` specified a list
+            of channels, `nbins` should be a list of integers. If `nbins`
+            is None, use ``FCSData.resolution(channel)``.
+        log : bool, optional
+            Whether to generate bins uniformly spaced in linear or
+            logarithmic scale.
+
+        Return
+        ------
+        array or list of arrays
+            Histogram bins for the specified channel(s).
+
+        """
```

Comment:
These are bin edges, as referred to in http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.histogram.html, where the `bins` parameter can contain "bin edges".

Fixed in 9151eb9a0ecda92820f041a943eb84b56ccfe5a2.